dockstore-webservice/src/main/java/io/dockstore/webservice/languages/NextflowHandler.java

```diff
@@ -542,7 +554,12 @@ private void getHelpMessage(GroovySourceAST definition, StringBuilder builder, S
             List<GroovySourceAST> processList = getGroovySourceASTList(mainDescriptor, "process");
 
             for (GroovySourceAST processAST : processList) {
-                String processName = getProcessValue(processAST);
+                String processName;
+                try {
+                    processName = getProcessValue(processAST);
+                } catch (NullPointerException e) {
```

Comment:
`getProcessValue` doesn't throw an NPE -- you catch it in there and return null. I think you need a null check here instead of a catch.

Or better, IMHO, have `getProcessValue` return an `Optional<String>`, then

```
getProcessValue(processAST).ifPresent(processName -> {
...
});
```

Reply:
Hmm...my latest changes didn't get commit/pushed for some reason